src/appsettings.json

```diff
@@ -6,5 +6,18 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "Email": {
+    "OurEmail": "smartsaverrecovery@gmail.com",
+    "EmailPassword": "Smartsaver123456",
+    "RecoveryMessageSubject": "Password Recovery",
+    "RecoveryMessageBody": "Your password recovery code is: ",
+    "GreetingMessageSubject": "Welcome to ePiggy!",
+    "GreetingMessageBody": "Thank you for joining ePiggy! Your wallet thanks you as well!",
+    "FarewellMessageSubject": "Farewell from the ePiggy team",
+    "FarewellMessageBody": "We're sorry to see you go, please leave your feedback of our app.",
+    "MaxRandValue": 999999,
+    "SmtpEmail": "smtp.gmail.com",
+    "Port": 587
+  },
```

Comment:
And as I understand when we read these as options we can change them during runtime and changes will be applied without restarting?

Reply:
I think that's how it works, but I tried for a bit and couldn't get it to work, so maybe I'm misunderstanding it